pkg/airgapped/plugin_bundle_download.go

```diff
@@ -244,8 +244,12 @@ func (o *DownloadPluginBundleOptions) downloadImagesAsTarFile(pluginEntries []*p
 // validateOptions validates the provided options and returns
 // error if contains invalid option
 func (o *DownloadPluginBundleOptions) validateOptions() error {
-	_, err := os.Stat(filepath.Dir(o.ToTar))
-	if err != nil {
+	// Check the input file path for --to-tar is valid
+	var empty []byte
+	err := os.WriteFile(o.ToTar, empty, 0644)
```

Comment:
I'm not a big fan of trying to write the file to check if the path is valid.
How about using `fileinfo, err := os.Stat` and then `fileinfo.IsDir()`?
Would that cover the cases you want to protect against?

Reply:
The reason I opted for the write-then-validate approach is to cover additional use cases beyond simply checking if the path is valid or represents a directory.
By writing a temporary file, we can also ensure that the input path has the necessary write permissions and is a valid path. 
While using `os.Stat` and `fileinfo.IsDir()` would cover the basic checks, they may not provide the same level of validation for other factors like path validation, write permissions and file system issues. By writing and then deleting the file, we can have a more comprehensive validation process.
Considering that we will be writing to the file after the plugins are downloaded, I believe this approach helps to proactively identify any potential issues before starting the download process.

Reply:
> By writing a temporary file, we can also ensure that the input path has the necessary write permissions and is a valid path.
> Considering that we will be writing to the file after the plugins are downloaded

Two good points.  Ok.  Sold 😄 .